src/SqlClient.Tests/SqlClient.Tests.fsproj

```diff
@@ -3,7 +3,7 @@
     <Import Project="../../netfx.props" />
     <Import Project="../../fsc.props" />
     <PropertyGroup>
-        <TargetFramework>netstandard20</TargetFramework>
+        <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
tests are apps, not libraries, so we need to make sure it targets as an app. We can/should also multitarget here to net462 in order to test the 462 version of the library.